accounts/static/accounts/js/onboarding_base.js

```diff
@@ -0,0 +1,34 @@
+const exit_modal = document.getElementById("confirm_exit_modal");
+const overlay = document.getElementById("exit_overlay");
+console.log("HELLO");
+function redirectToLandingPage() {
```

Comment:
can't these just be like 'onClick' function lambdas?
the declaration is longer than the actual code itself.